zulip_bots/zulip_bots/bots/xkcd/doc.md

```diff
@@ -10,7 +8,7 @@ like so:
 
 Run this bot as described in [here](https://zulipchat.com/api/running-bots#running-a-bot).
 
-## Usage
+### Usage
```

Comment:
What's the story with this change?

Reply:
All the other pages that have subtitles like this one, have <h3> instead of <h2>. I thought they should all have the same type of heading.
These are some of the pages that have subtitles:
https://zulip.com/integrations/doc/matrix
https://zulip.com/integrations/doc/freshdesk
https://zulip.com/integrations/doc/irc
https://zulip.com/integrations/doc/jenkins